app/views/content/is-teaching-right-for-me/maths/maths/_article.html.erb

```diff
@@ -0,0 +1,95 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      If maths is your passion, you'll know that it's not just learning formulas or solving equations. It's a way of thinking and problem solving that’s important in every walk of life.
+    </p>
+
+    <p>
+      An exciting career spent passing your knowledge on to the next generation could be for you.
+    </p>
+
```

Comment:
@caitlin-reidy do you think it's worth keeping in the line about £30k for bursaries and scholarships in the intro? Marios said in UR feedback was that it could be a deciding factor to someone choosing maths. So it might be good to mention at the start to encourage people to read on?